docs/sw-update-conf.json

```diff
@@ -102,6 +107,11 @@
           "repo": "iobeam",
           "pip": "git+ssh://git@bitbucket.org/mrbeam/iobeam.git@{target_version}",
           "global_pip_command": true,
+          "beamos_version": {
+            "0.18.0": {
+              "pip_command": "sudo /usr/local/iobeam/venv/bin/pip"
+            }
+          },
           "beamos_date": {
```

Comment:
why we still keep the date ?

Reply:
I can remove it but this means to have a new major version of the config and a change in the plugin to use the new major version